beta/nncf/tensorflow/graph/model_transformer.py

```diff
@@ -144,7 +144,7 @@ def _get_target_layer_name(self, target_point):
     def _insert(self, target_point, insertion_objects):
         target_layer_name = self._get_target_layer_name(target_point)
 
-        if target_point.type == TargetType.WEIGHT_OPERATION:
+        if target_point.type == TargetType.LAYER_WEIGHT_OPERATION:
```

Comment:
What is the intent of renaming this? To me, OPERATION_WITH_WEIGHTS sound more meaningful.

Reply:
I agree, thanks. I have renamed it.